### DISCUSSION TITLE
Exclude/hide transaction amounts from reports/balance

### DISCUSSION THREAD

**Discussion by Aohzan** [firefly-iii/firefly-iii#6815] [Firefly III]
Hello,

When the bank unlock a part of a credit and made a transfer after that, you have something like in your transactions:
+ 10 000€ unlock from bank
- 10 000€ transfer to pay

Is it possible to exclude or hide those transaction with a tag/category/something else to not appears in reports and the account balance ?
Optionally I can delete them, but with the risk that they come again with the next import, and I lost the history

Thank you

**Comment by JC5** [comment]
This will not be possible unfortunately, but if you delete the transaction it will *not* be imported again in the future.
